ScaleformUI_Lua/src/PauseMenu/TabView.lua

```diff
@@ -880,7 +880,7 @@ function TabView:ProcessControl()
             self:GoLeft()
         end)
     end
-    if (IsControlJustPressed(2, 175)) then
+    if (IsControlJustPressed(2, 175) or IsControlJustPressed(2, 192)) then
```

Comment:
I'd add a check for this to be used only on keyboard.. pressing Y to change tab on controller seems odd